src/mongocxx/test/spec/crud.cpp

```diff
@@ -59,107 +59,110 @@ void run_crud_tests_in_file(std::string test_path) {
     }
 
     for (auto&& test : test_spec_view["tests"].get_array().value) {
-        INFO("Test description: " << test["description"].get_utf8().value);
-
-        if (is_unsupported(test["description"].get_utf8().value)) {
-            continue;
-        }
-
-        if (should_skip_spec_test(client, test.get_document())) {
-            continue;
-        }
+        auto description = test["description"].get_utf8().value;
+        SECTION(to_string(description)) {
```

Comment:
Everything below this (excluding line 91, which I changed) stayed the same. It's just indented.